lib/solidus_multi_domain/multi_domain_helpers.rb

```diff
@@ -16,7 +15,12 @@ module MultiDomainHelpers
     end
 
     def get_taxonomies
-      @taxonomies ||= current_store.present? ? ::Spree::Taxonomy.where(["store_id = ?", current_store.id]) : ::Spree::Taxonomy
+      @taxonomies ||= if current_store.present?
+                        ::Spree::Taxonomy.where(["store_id = ?",
+                                                 current_store.id])
+                      else
+                        ::Spree::Taxonomy
+                      end
       @taxonomies = @taxonomies.includes(root: :children)
       @taxonomies
     end
```

Comment:
I think this is the method that is available in the new starter frontend through the taxon helper.